scripts.py

```diff
@@ -1262,40 +1373,38 @@ def extract_feed_configuration(cls, feed_config, parsed_args, uploader=None):
             if not lanes_policy:
                 raise cls.IncompleteFeedConfigurationError("No LANES_POLICY found")
 
-            enabled_facets = C.policy(C.FACET_POLICY)
+            enabled_facets = C.policy(u'facets')
 
             if not license_link:
                 # Attempt to get the link this feed's license from the
                 # config file if it wasn't included on the command line.
-                license_link = C.link(C.LICENSE)
+                license_link = C.get('links', {}).get('license')
 
-            def confirm_configuration(configuration_dict, expected_keys, name):
-                """Ensures that the configuration is included for any fields
-                required for static feed generation.
-                """
-                missing = filter(lambda k: k not in configuration_dict, expected_keys)
-                if missing:
-                    missing = ["'%s'" % k for k in missing]
-                    missing = ", ".join(missing)
+            if parsed_args.upload and not uploader:
+                s3_config = C.integration('S3')
+                access_key = s3_config.get('access_key')
+                secret_key = s3_config.get('secret_key')
 
+                if not (access_key and secret_key):
                     raise cls.IncompleteFeedConfigurationError(
-                        "Incomplete %s configuration: missing %s" % (
-                        name, missing))
+                        'Incomplete S3 configuration'
+                    )
 
-            if parsed_args.upload and not uploader:
-                s3_integration = C.integration(C.S3_INTEGRATION)
-                confirm_configuration(
-                    s3_integration, cls.S3_CONFIG_KEYS, C.S3_INTEGRATION
-                )
-                uploader = S3Uploader()
+                uploader = S3Uploader(access_key, secret_key)
+
+            search_config = C.integration('Elasticsearch')
+            if search_config:
+                url = search_config.get('url')
+                works_index = search_config.get(ExternalSearchIndex.WORKS_INDEX_KEY)
```

Comment:
Eventually saving External Integrations for generating certain static feeds might make sense, but for now we are passing in an external file that mimics the (former) Configuration files (RIP).